src/grid-list/grid-list.jsx

```diff
@@ -101,6 +101,7 @@ const GridList = React.createClass({
     const mergedRootStyles = this.mergeStyles(styles.root, style);
 
     const wrappedChildren = React.Children.map(children, (currentChild) => {
+      if (currentChild.type.displayName === 'Subheader') return currentChild;
```

Comment:
`currentChild` as `currentChild.type` can be null or undefined. What about using `React.isValidElement` like in TableRow or ```js
      if (!currentChild) {
        return null;
      }
      if (!currentChild.type) {
        return currentChild;
      }
``` like in ToolbarGroup?